### PR TITLE
docs(GiniCaptureSDK): Analysis screen customization guide

### PR DESCRIPTION
**Pull request by zladzeyka**
- Remove old customization options, added Figma links
- Added Customization guide for the analysis screen
- Added TODOs

PIA-2296

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]
Nice!

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md [hunk 1]
```diff
@@ -112,105 +118,32 @@ During QR Code only mode the capture and import controls will be hidden from the
...
-	  - Image &#8594; <span style="color:#009EDF">*navigationAnalysisBack*</span> image asset
-	  - Title &#8594; <span style="color:#009EDF">*ginicapture.navigationbar.analysis.back*</span> localized string
-  - With title only
-	  - Title &#8594; `GiniConfiguration.navigationBarAnalysisTitleBackButton`
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2F1985HMF83siAXmysSn3dC6%2FiOS-Gini-Capture-SDK-3.0.0-UI-Customisation%3Fnode-id%3D501%253A7494%26t%3DcRAvcUKVlwGtGpuh-1" allowfullscreen></iframe>
 
-##### 2. PDF Information view
-- Text color &#8594; `GiniConfiguration.analysisPDFInformationTextColor`
-- Background color &#8594; `GiniConfiguration.analysisPDFInformationBackgroundColor`
+You can show a custom loading indicator with custom animation support.
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:139]: [Suggestion] Maybe we should mention here that the same loading indicator will be also used when a correct QR code is scanned.

**Review by vizaknaidavid — APPROVED** [review]
Looks good. A tiny suggestion, but consider it accepted even without that. :D

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md (hunk 1 above)
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:142]: Added it for QR code scanning 😉

**Pull request merged**